data/ext/pending/issue-1909.rdfa

```diff
@@ -0,0 +1,9 @@
+<div>
+  <!-- course workload, issue 1909 -->
+  <div typeof="rdf:Property" resource="http://schema.org/courseWorkload">
+      <span class="h" property="rdfs:label">courseWorkload</span>
+      <span property="rdfs:comment">The amount of work expected of students on the course, often provide as a figure per week or per month, and may be broken down by type. For example, "2 hours of lectures, 1 hour of lab work and 3 hours of independent study per week".</span>
```

Comment:
"often provide**d**"

Reply:
Thanks, fixed that along with a change suggested after comment by @stuartasutton